dags/hivemind_etl_helpers/src/db/discord/utils/fetch_channel_thread_name.py

```
@@ -0,0 +1,25 @@
+        self.client = MongoSingleton.get_instance().get_client()
+        self.db = self.client[guild_id]
+        self.channels_collection = self.db["channels"]
+        self.threads_collection = self.db["threads"]
+
+    def fetch_discord_channel_name(self, channel_id: str) -> str:
+        query = {"channelId": channel_id}
+        channel = self.channels_collection.find_one(query)
+        if channel is None:
+            return None
+        else:
+            return channel["name"]
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)